Treppenhauslichtsteuerung/module.php

```diff
@@ -143,6 +155,24 @@ public function GetConfigurationForm()
             ];
         }
 
+        $hideNightModeElements = function (array $states) use (&$jsonForm)
```

Comment:
Das wirkt super kompliziert. Ich verstehe, dass du wahrscheinlich die Logik in einer Funktion haben wolltest, sodass man diese nicht "dupliziert", aber dadurch ist der Code recht komplex. Ich dachte eher an sowas:

`$jsonForm['elements'][3]['items'][1]['visible'] = in_array($nightMode, ['off', 'integer']);`

Hast du evtl. noch eine Idee wir man es einfacher machen könnte?